src/home_robot/home_robot/utils/point_cloud_torch.py

```diff
@@ -247,4 +247,10 @@ def get_bounds(points: Tensor):
     Returns:
         mins_and_maxes: [3, 2]
     """
-    return torch.stack([points.min(dim=0)[0], points.max(dim=0)[0]], dim=-1)
+    mins = points.min(dim=0)[0]
+    maxs = points.max(dim=0)[0]
+    if not torch.all(maxs > mins):
```

Comment:
Maybe add a `warnings.warn(RuntimeError("Zero-volume bounds detected in get_bounds"))`

Reply:
done